lib/helper.py

```diff
@@ -89,18 +89,20 @@ def get_dist():
 
 def get_machine_type():
     """
-    Return What kind of machine example: pHyp/PowerNV
+    Return What kind of machine example: pHypLpar/PowerNV/qemu
     """
     machine_type = None
     cpuinfo = '/proc/cpuinfo'
-    if os.path.isfile(cpuinfo):
```

Comment:
Why remove this check ?

Reply:
no harm adding it back though, 
but, that check was kind of unness, I doubt if we have `/proc/cpuinfo'  missing in any linux env. what say?